packages/editor-ui/src/components/Node.vue

```diff
@@ -16,6 +16,11 @@
 				</n8n-tooltip>
 			</div>
 
+			<n8n-tooltip :placement="'top'" :manual="true" :open-delay="2500" :value="showWebhookNodeTooltip">
+				<div slot="content" v-html="`Waiting for you to create an event in ${getTrimmedWebhookNodeName()}`"></div>
```

Comment:
<img width="634" alt="Screen Shot 2021-11-12 at 12 27 57 PM" src="https://user-images.githubusercontent.com/4711238/141460030-07672929-fdb6-48bc-9661-3858776041a4.png">
when panning or zooming out, tooltip position is not updated.. we need to fix this somehow or just hide the tooltip

Reply:
This is fixed. I put watchers to observe if the canvas change its place, and if WF is zoomed in/out.